### PR TITLE
crimson/net: print tx/rx messages using logger().info()

### PR DESCRIPTION
**Pull request by tchaikov**
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/crimson/net/ProtocolV2.cc [hunk 1]
```diff
@@ -1384,8 +1384,9 @@ ceph::bufferlist ProtocolV2::do_sweep_messages(
 
     auto message = MessageFrame::Encode(header2,
         msg->get_payload(), msg->get_middle(), msg->get_data());
-    logger().debug("{} write msg type={} off={} seq={}",
-                   conn, header2.type, header2.data_off, header2.seq);
+    logger().info("{} --> [{} {}] #{} === {} ({}) // {}",
+		  messenger, peer_name, conn.get_peer_addr(),
+		  msg->get_seq(), *msg, msg->get_type(), conn);
```
Comment (cyx1231st) [inline src/crimson/net/ProtocolV2.cc]: `info` logs will be printed by default, so this means we need to change default log level to >= `warn` for perf tests.

Comment (tchaikov) [reply src/crimson/net/ProtocolV2.cc]: @cyx1231st i see. how about moving all the logging messages in messenger to trace level ?

Comment (cyx1231st) [reply src/crimson/net/ProtocolV2.cc]: Agree, crimson messenger haven't utilized trace level logs before.

**Comment by tchaikov** [comment]
@cyx1231st could you please take another look?

**Comment by tchaikov** [comment]
retest this please.

**Pull request merged**